base/src/META-INF/blaze-base.xml

```diff
@@ -409,6 +410,9 @@
     <registryKey defaultValue="true"
                  description="Causes the plugin to read external workspace data, enabling features such as code completion for external targets. This may incur the cost of an additional 'bazel mod' call, which could cause issues with older versions or setups."
                  key="bazel.read.external.workspace.data"/>
+    <registryKey defaultValue="true"
+                 description="Enable basic syntax highliting in Query Sync's 'Analysis Disabled' mode"
+                 key="bazel.qsync.enable.basic.highlighting.when.analysis.disabled"/>
```

Comment:
For flags trailing `.disabled` can be confusing... Usually it is attributed to flag default state.

Reply:
right!